code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -6,14 +6,14 @@
 	actions_types = list(/datum/action/item_action/hands_free/activate)
 	var/activated = 1 //1 for implant types that can be activated, 0 for ones that are "always on" like mindshield implants
 	var/implanted = null
-	var/mob/living/imp_in = null
+	var/mob/living/carbon/imp_in = null

```

Comment:
I believe implants in simple animals are intended ? Or is there some code that prevents that anyway ?